manticore/platforms/evm.py

```diff
@@ -608,6 +644,16 @@ def __setstate__(self, state):
         super().__setstate__(state)
 
     def _get_memfee(self, address, size=1):
+        """
+            This calculates the ammount of extra gas needed for accessing to
```

Comment:
```suggestion
            This calculates the amount of extra gas needed for accessing to
```
Also, can we remove the additional indent made here?